--- origsrc/gle-3.1.0/Makefile.am	2001-07-20 17:06:59.000000000 -0500
+++ src/gle-3.1.0/Makefile.am	2007-05-25 16:32:23.323601400 -0500
@@ -5,6 +5,6 @@
   .cvsignore \
   gle.spec
 
-docdir = $(datadir)/doc/gle/
+docdir = $(datadir)/doc/gle-$(VERSION)
 
 doc_DATA = AUTHORS README COPYING
--- origsrc/gle-3.1.0/configure.in	2003-03-17 22:22:16.000000000 -0600
+++ src/gle-3.1.0/configure.in	2007-05-25 15:26:55.085820600 -0500
@@ -134,7 +134,7 @@
 #       cc @X_LIBS@ (-lfoo...) @X_PRE_LIBS@ -lX11 @X_EXTRA_LIBS@
 # (Perhaps X_LIBS should have been called X_LDFLAGS.)
 
-X_LIBS="-lgle $X_LIBS -lXt -lX11"
+X_LIBS="\$(top_builddir)/src/libgle.la $X_LIBS -lXt -lX11"
 
 AC_OUTPUT(
           Makefile
@@ -146,5 +146,5 @@
           ms-visual-c/Makefile
           ms-visual-c/gle/Makefile
           swig/Makefile,
-          [ln -sf src GL]
+          [ln -sf $srcdir/src GL]
           )
--- origsrc/gle-3.1.0/doc/Makefile.am	2001-11-21 00:32:55.000000000 -0600
+++ src/gle-3.1.0/doc/Makefile.am	2007-05-25 16:24:27.833140200 -0500
@@ -1,7 +1,7 @@
 
 SUBDIRS = html
 
-docdir = $(datadir)/doc/gle/
+docdir = $(datadir)/doc/gle-$(VERSION)
 
 doc_DATA =  COPYING.artistic gle-3.0.lsm
 
--- origsrc/gle-3.1.0/doc/html/Makefile.am	2001-07-20 15:51:43.000000000 -0500
+++ src/gle-3.1.0/doc/html/Makefile.am	2007-05-25 16:24:17.567383800 -0500
@@ -81,7 +81,7 @@
   twist2.orig.gif	\
   twist3.orig.gif
 
-docdir = $(datadir)/doc/gle/html
+docdir = $(datadir)/doc/gle-$(VERSION)/html
 
 doc_DATA =  $(html_HTML) $(gif_HTML) $(giforig_HTML) README
 
--- origsrc/gle-3.1.0/examples/Makefile.am	2003-03-17 22:18:55.000000000 -0600
+++ src/gle-3.1.0/examples/Makefile.am	2007-05-25 16:32:12.073457400 -0500
@@ -58,7 +58,7 @@
 
 LDADD = @X_LIBS@  -L../src/
 
-docdir = $(datadir)/doc/gle/examples
+docdir = $(datadir)/doc/gle-$(VERSION)/examples
 
 doc_DATA = \
    COPYING \
--- origsrc/gle-3.1.0/src/Makefile.am	2003-03-17 22:09:44.000000000 -0600
+++ src/gle-3.1.0/src/Makefile.am	2007-05-25 15:26:55.101445800 -0500
@@ -21,7 +21,8 @@
   view.c
 
 
-libgle_la_LDFLAGS = -version-info 4:0:1
+libgle_la_LDFLAGS = -version-info 4:0:1 -no-undefined
+libgle_la_LIBADD = -lGLU -lGL
 
 # make sure we install gle.h into /usr/include/GL/gle.h or thereabouts
 pkgincludedir = $(includedir)/GL
@@ -51,7 +52,7 @@
 
 CFLAGS = @CFLAGS@ -Wall -pedantic -Wstrict-prototypes  -Wconversion
 
-docdir = $(datadir)/doc/gle/
+docdir = $(datadir)/doc/gle-$(VERSION)
 
 doc_DATA = COPYING.src
 
