diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/gle-3.1.0/Makefile.am src/gle-3.1.0/Makefile.am
--- origsrc/gle-3.1.0/Makefile.am	2001-07-20 17:06:59.000000000 -0500
+++ src/gle-3.1.0/Makefile.am	2007-05-25 16:32:23.323601400 -0500
@@ -5,6 +5,6 @@
   .cvsignore \
   gle.spec
 
-docdir = $(datadir)/doc/gle/
+docdir = $(datadir)/doc/gle-$(VERSION)
 
 doc_DATA = AUTHORS README COPYING
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/gle-3.1.0/configure.in src/gle-3.1.0/configure.in
--- origsrc/gle-3.1.0/configure.in	2003-03-17 22:22:16.000000000 -0600
+++ src/gle-3.1.0/configure.in	2007-05-25 15:26:55.085820600 -0500
@@ -134,7 +134,7 @@
 #       cc @X_LIBS@ (-lfoo...) @X_PRE_LIBS@ -lX11 @X_EXTRA_LIBS@
 # (Perhaps X_LIBS should have been called X_LDFLAGS.)
 
-X_LIBS="-lgle $X_LIBS -lXt -lX11"
+X_LIBS="\$(top_builddir)/src/libgle.la $X_LIBS -lXt -lX11"
 
 AC_OUTPUT(
           Makefile
@@ -146,5 +146,5 @@
           ms-visual-c/Makefile
           ms-visual-c/gle/Makefile
           swig/Makefile,
-          [ln -sf src GL]
+          [ln -sf $srcdir/src GL]
           )
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/gle-3.1.0/doc/Makefile.am src/gle-3.1.0/doc/Makefile.am
--- origsrc/gle-3.1.0/doc/Makefile.am	2001-11-21 00:32:55.000000000 -0600
+++ src/gle-3.1.0/doc/Makefile.am	2007-05-25 16:24:27.833140200 -0500
@@ -1,7 +1,7 @@
 
 SUBDIRS = html
 
-docdir = $(datadir)/doc/gle/
+docdir = $(datadir)/doc/gle-$(VERSION)
 
 doc_DATA =  COPYING.artistic gle-3.0.lsm
 
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/gle-3.1.0/doc/html/Makefile.am src/gle-3.1.0/doc/html/Makefile.am
--- origsrc/gle-3.1.0/doc/html/Makefile.am	2001-07-20 15:51:43.000000000 -0500
+++ src/gle-3.1.0/doc/html/Makefile.am	2007-05-25 16:24:17.567383800 -0500
@@ -81,7 +81,7 @@
   twist2.orig.gif	\
   twist3.orig.gif
 
-docdir = $(datadir)/doc/gle/html
+docdir = $(datadir)/doc/gle-$(VERSION)/html
 
 doc_DATA =  $(html_HTML) $(gif_HTML) $(giforig_HTML) README
 
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/gle-3.1.0/examples/Makefile.am src/gle-3.1.0/examples/Makefile.am
--- origsrc/gle-3.1.0/examples/Makefile.am	2003-03-17 22:18:55.000000000 -0600
+++ src/gle-3.1.0/examples/Makefile.am	2007-05-25 16:32:12.073457400 -0500
@@ -58,7 +58,7 @@
 
 LDADD = @X_LIBS@  -L../src/
 
-docdir = $(datadir)/doc/gle/examples
+docdir = $(datadir)/doc/gle-$(VERSION)/examples
 
 doc_DATA = \
    COPYING \
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/gle-3.1.0/src/Makefile.am src/gle-3.1.0/src/Makefile.am
--- origsrc/gle-3.1.0/src/Makefile.am	2003-03-17 22:09:44.000000000 -0600
+++ src/gle-3.1.0/src/Makefile.am	2007-05-25 15:26:55.101445800 -0500
@@ -21,7 +21,8 @@
   view.c
 
 
-libgle_la_LDFLAGS = -version-info 4:0:1
+libgle_la_LDFLAGS = -version-info 4:0:1 -no-undefined
+libgle_la_LIBADD = -lGLU -lGL
 
 # make sure we install gle.h into /usr/include/GL/gle.h or thereabouts
 pkgincludedir = $(includedir)/GL
@@ -51,7 +52,7 @@
 
 CFLAGS = @CFLAGS@ -Wall -pedantic -Wstrict-prototypes  -Wconversion
 
-docdir = $(datadir)/doc/gle/
+docdir = $(datadir)/doc/gle-$(VERSION)
 
 doc_DATA = COPYING.src
 
